scenes/game/local_game.gd

```diff
@@ -5462,7 +5825,7 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			assert(false, "ERROR: Unhandled effect type: %s" % effect['effect_type'])
 
 	if not ignore_extra_effects:
-		if "and" in effect:
+		if "and" in effect and effect['and']:
```

Comment:
avoiding setting "and" to null would probably be safer, I think there's at least one other place where I did weird things with the and effect (character actions?)

Reply:
I think it's easier to let us set it to null and just check it here